### ISSUE TITLE
Ability to change Silo size and timeout

### ISSUE THREAD

**Issue by ealymbaev** [emqtt/CocoaMQTT#154]
Hi, recently I had an issue that my socket was frequently disconnected before I receive Publish Ack back. And after some time (quite short) my socket client could not publish anything anymore.

After researching the library code I have found out that the silo was filled very quickly and after that  the publish (send) code was not performed anymore (tryTransport() function).

I think that default silo size (10) is too small and timeout is too long (60 seconds).

It would be great if I could config them myself when creating client.

**Comment by alexyu1989** [comment]
Done. 
Try use the latest code on GitHub. (not the cocoapods release version.) 

`pod 'CocoaMQTT', :git => 'https://github.com/emqtt/CocoaMQTT.git'`

```
mqtt.bufferSilosTimeout = 20
mqtt.bufferSilosMaxNumber = 50
```

**Comment by alexyu1989** [comment]
@ealymbaev Plz reopen this issue if you have any problems.

**Comment by ealymbaev** [comment]
Thanks, when are you planning to release next version?

**Comment by alexyu1989** [comment]
already released 1.0.19

**Comment by zl00** [comment]
:thumbsup:
